joomla/joomla-cms
# Don't enable sending the PW on new installs

### Summary of Changes

With this PR we switch the default to not send the plain PW via mail. Similar too all services out there not sending the PW out too.

As discussed with the JSST this is now a public tracker item: cc @SniperSister @mbabker 

### Testing Instructions

Install https://github.com/zero-24/joomla-cms/archive/disable_sending_plainpw.zip
make sure the sending PW option is disabled
make sure the pw field now is required for new users
enable the setting to send PW
make sure the pw field is now optional for new users

### Expected result

Sending PW via Mail is disabled by default

### Actual result

Sending PW via Mail is enabled by default
